api/src/main/java/com/exelaration/abstractmemery/security/WebSecurity.java

```diff
@@ -51,6 +52,8 @@ protected void configure(HttpSecurity http) throws Exception {
         .authorizeRequests()
         .antMatchers(HttpMethod.POST, SIGN_UP_URL)
         .permitAll()
+        .antMatchers(HttpMethod.PUT, SETTINGS_URL)
```

Comment:
Why was this added to the antMatchers? Shouldn't the user be logged in if they are looking at their settings?

Reply:
Pretty sure I forgot to take this out when I was using it to debug. Nice catch I'll remove it now